src/panels/draft/DraftValidatePanel.ts

```diff
@@ -0,0 +1,73 @@
+import { Uri, WorkspaceFolder, window } from "vscode";
+import path from "path";
+import { BasePanel, PanelDataProvider } from "../BasePanel";
+import {
+    InitialState,
+    ToVsCodeMsgDef,
+    ToWebViewMsgDef,
+} from "../../webview-contract/webviewDefinitions/draft/draftValidate";
+import { TelemetryDefinition } from "../../webview-contract/webviewTypes";
+import { MessageHandler, MessageSink } from "../../webview-contract/messaging";
+import { ShellOptions, exec } from "../../commands/utils/shell";
+import { failed } from "../../commands/utils/errorable";
+
+export class DraftValidatePanel extends BasePanel<"draftValidate"> {
+    constructor(extensionUri: Uri) {
+        super(extensionUri, "draftValidate", {
+            validationResult: undefined,
+        });
+    }
+}
+
+export class DraftValidateDataProvider implements PanelDataProvider<"draftValidate"> {
+    readonly draftDirectory: string;
+    constructor(
+        readonly workspaceFolder: WorkspaceFolder,
+        readonly draftBinaryPath: string,
+        readonly initialLocation: string,
+    ) {
+        this.draftDirectory = path.dirname(draftBinaryPath);
+    }
+
+    getTitle(): string {
+        return `Draft Validate in ${this.workspaceFolder.name}`;
+    }
+
+    getInitialState(): InitialState {
+        return {
+            validationResults: "Initializing validation...",
+        };
+    }
+
+    getTelemetryDefinition(): TelemetryDefinition<"draftValidate"> {
```

Comment:
That's to record the usage data. Let me set it to true to see future usage. Had set it to false during testing, to prevent exponential false usage. Thanks!